htsget-search/CHANGELOG.md

```diff
@@ -0,0 +1,153 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.1.1](https://github.com/umccr/htsget-rs/compare/htsget-search-v0.1.0...htsget-search-v0.1.1) - 2023-01-27
+
+### Fixed
+- fixes for byte range ends
+- fix IOError name
+- fixes & model improvements & bam search progress
+- fixes & model improvements & bam search progress
+- fix docs
+
+### Other
```

Comment:
Does this include every commit from the whole repo? Interesting how this is here, even though most of these commits don't follow the conventional commit format. I guess that's why it's under the `Other` heading.

Do we want everything to be present from the beginning as part of our initial commit? @brainstorm

Reply:
Hi, I'm release-plz author! I was looking around because of a bug report from brainstorm :)
> Does this include every commit from the whole repo?

The changelog of each crate includes the commit that edited one of the files of the crate or a one of its dependencies.

> I guess that's why it's under the Other heading.

Yes!

Reply:
> The changelog of each crate includes the commit that edited one of the files of the crate or one of its dependencies.

Ahh okay, that makes sense. Thanks @MarcoIeni!

Reply:
I just noticed as well, `htsget-search` is the only crate that has the full history of it's commits because it's the only crate that hasn't been [renamed recently](https://github.com/umccr/htsget-rs/pull/98/commits/a017b88ef9411597b628749777d4cd5a8daad747).